src/riscv.h

```diff
@@ -42,6 +42,17 @@ enum {
     rv_reg_t6,
 };
 
+/* RISC-V exception code list */
+enum {
```

Comment:
Move the enumeration to C source.

Reply:
With the [X macro](https://en.wikipedia.org/wiki/X_Macro) trick, we can write down shorter code.
Check https://github.com/sysprog21/rv32emu/blob/master/src/syscall.c (SUPPORTED_SYSCALLS)